src/Akka.Persistence.Sql/Db/AkkaPersistenceDataConnectionFactory.cs

```diff
@@ -139,6 +143,13 @@ public AkkaPersistenceDataConnectionFactory(IProviderConfig<SnapshotTableConfigu
                 .Member(r => r.TagArr)
                 .IsNotColumn();
 
+            if (config.ProviderName.ToLower().Contains("sqlite"))
+            {
+                journalRowBuilder
+                    .Member(r => r.Ordering)
+                    .HasDbType("INTEGER");
```

Comment:
Is this a 64 bit or 32 bit integer?

Reply:
I know how it's how we've been doing this previously, but I wanted to understand - we should also add a comment to the code here explaining why this is necessary for Sqlite.

Reply:
To be precise, Sqlite only recognizes 5 data types: "NULL", "INTEGER", "REAL", "TEXT", "BLOB"

Reply:
TY for the explanation